sirepo/package_data/template/srw/README.txt.jinja

```diff
@@ -0,0 +1,35 @@
+These files facilitate the generation and use of input data for machine learning for SRW. They include:
+
+{{ shFileName }}: a shell script that generates input data
+{{ pyFileName }}: a python script that runs SRW using the generated data
+{{ ymlFileName }}: a YML configuration file for rsopt
+{{ readmeFileName }}: this README
+
+{% if hasLibFiles %}
+Also included are auxiliary files required for the specific SRW simulation:
+{{ libFiles }}
+{% endif %}
+
+To generate the data, run 'bash {{ shFileName }}'. This stores the data in the numpy file H_sample_{{ fileBase }}_history_length={{ totalSamples }}_evals={{ totalSamples }}_workers={{ numWorkers }}.npy
+The script will prompt you to then run SRW with that data. To do so later, run 'python {{ pyFileName }} rsopt_run H_sample_{{ fileBase }}_history_length={{ totalSamples }}_evals={{ totalSamples }}_workers={{ numWorkers }}.npy'. Please note:
```

Comment:
This line is too long, break it up so the "python ..." is on its own line.

Reply:
41a07e1